website/pages/auto-import-components.mdx

```diff
@@ -13,16 +14,15 @@ This provides a better developer experience without sacrificing your application
 
 
 ## Installation
-With Yarn:
 
-```bash
-yarn add chakra-loader -D
-```
-With NPM:
-
-```bash
-npm install chakra-loader --save-dev
-```
+<CodeGroup lang="bash">
+  <CodeTab label="Yarn">
+    yarn add chakra-loader
+  </CodeTab>
+  <CodeTab label="NPM">
+    npm install chakra-loader
+  </CodeTab>
+</CodeGroup>
```

Comment:
Since `chakra-loader` is a development dependency, we may need to include the flag `-D` for yarn and `--save-dev` for npm

Reply:
Good catch, I will update those!